analytics_data_api/management/commands/generate_stage_course_data.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable=line-too-long,invalid-name
+
+import datetime
+import logging
+
+from analytics_data_api.management.commands.generate_data import FakeDataCommand
+from analytics_data_api.v0 import models
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class Command(FakeDataCommand):
+    help = 'Generate fake data for stage environment'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--database',
+            action='store',
+            dest='database',
+            default='default',
+            help='Database in which to generate fake date',
+        )
+
+    def get_start_date(self, database):
+        enrollments = models.CourseEnrollmentDaily.objects.using(database)
+        # default to ten weeks of data if no data exists
+        start_date = datetime.datetime.now() - datetime.timedelta(weeks=4)
+        if enrollments:
+            latest_date = enrollments.latest().date
+            # want to return latest date + some time difference to avoid overlap
+            start_date = datetime.datetime.combine(
+                latest_date + datetime.timedelta(days=1), datetime.datetime.min.time()
```

Comment:
Start adding data a day after our last enrollment record date, that way we can continually add data each week